addons/project/static/src/js/project_calendar.js

```diff
@@ -0,0 +1,15 @@
+odoo.define('project.ProjectCalendarController', function (require) {
+"use strict";
+const CalendarController = require('web.CalendarController');
+
+
+const ProjectCalendarController = CalendarController.include({
+    _renderButtonsParameters() {
+        let buttonsParams = this._super.apply(this, arguments);
+        if (this.model.modelName === 'project.task') {
+            buttonsParams = {scaleDrop: true};
+        }
+        return _.extend({}, this._super.apply(this, arguments), buttonsParams);
```

Comment:
Same here

Reply:
@kba-odoo  Thanks, I also removed the upper super call because this extend already get the parameters and concatenate them with the new value.